readme.md

```diff
@@ -52,7 +52,7 @@ Type: `Array`
 
 ### cycled
 
-The instance is an iterable that will cycle through the array indefinitely.
+The instance is an iterable that will cycle through the array.
```

Comment:
Needs to be more explicit about how it works. It will cycle through the number of elements equaling the length of the array from the current index.